### ISSUE TITLE
Feature request: Capture wrong pillar.get

### ISSUE THREAD

**Issue by jbouter** [warpnet/salt-lint#57]
I'd like to see a function implemented where usage of wrong `pillar.get` is detected and reported.

For example:
**wrong**:
```code
pillar.get('pillar:item')
```

**good**:
```code
salt['pillar.get']('pillar:item')
```

```code
pillar['pillar']['item']
```

This limits itself to state files, above "wrong" examples would most likely still work in pillarstack.

**Comment by manangoel99** [comment]
I'd like to take up this issue

**Comment by roaldnefs** [comment]
@manangoel99, thanks for your interest in the project!

This issue and #58 are already fixed in #60 by @jbouter.
